user-guide/Troubleshooting/RemoteLogCollection.md

```diff
@@ -0,0 +1,18 @@
+---
+uid: RemoteLogCollection
+---
+
+# Remote Log Collection
+
+## What is it?
+
+The Remote Log Collection feature will allow our experts to instantly gather data from all nodes in your system. The ultimate goal is to enable our them to gather the necessary information from your system without your intervention. With this functionality experts from Skyline can trigger the LogCollector tool remotely, so they can collect the logs and memory dumps they need for their investigation.
```

Comment:
feature allows our .. iso will allow
It feels like the same thing is being repeated 3 times but formulated differently.
Should stick to same formatting as done in other modules , aka xxxx module ...

The Remote Log Collection module allows Skyline Techsupport employees (insert correct correct title for techsupport) to collect DataMiner log and memory dump files without customer assistance,  increasing efficiency of investigations.

Reply:
Remote Log Collection is really a feature and not a module IMO.